code/modules/events/aurora_caelus.dm

```diff
@@ -32,6 +32,20 @@
 		if(A.area_flags & AREA_USES_STARLIGHT)
 			for(var/turf/open/space/S in A)
 				S.set_light(S.light_range * 3, S.light_power * 0.5)
+		if(istype(A, /area/station/service/kitchen))
+			for(var/turf/open/kitchen in A)
+				kitchen.set_light(1, 0.75)
+			if(prob(1) || SSevents.holidays?[APRIL_FOOLS])
+				var/obj/machinery/oven/roast_ruiner = locate() in A
+				if(roast_ruiner)
+					roast_ruiner.balloon_alert_to_viewers("oh egads!")
+					var/turf/ruined_roast = get_turf(roast_ruiner)
+					ruined_roast.atmos_spawn_air("plasma=100;TEMP=1000")
+				for(var/mob/living/carbon/human/seymour in GLOB.human_list)
```

Comment:
add an `as anything`, human list should only have humans

Reply:
i thought as anything ran typechecks